etsin_finder/services/common_service.py

```diff
@@ -94,9 +99,36 @@ def _get_args(self, **kwargs):
         args.update(kwargs)
         return args
 
-    def get_directory(self, dir_identifier, params=None):
+    def get_directory_for_project_using_scope(self, cr_id, pid, scope, params=None):
```

Comment:
I'd prefer `path` instead of `scope` for this function since that's what the Metax parameter is called.